dev/parm/config/gfs/config.ecen_fv3jedi

```diff
@@ -21,7 +21,7 @@ export STAGE_YAML="${PARMgfs}/gdas/atm/atm_ecen_stage.yaml.j2"
 if [[ ${DOHYBVAR} = "YES" ]]; then
     export CASE_ANL=${CASE_ENS}
 else
-    export CASE_ANL=${CASE}
+    export CASE_ANL=${CASE_HIST}
```

Comment:
I guess, the ufsda test will reveal the above question (may be).  @DavidNew-NOAA would know.